README.md

```diff
@@ -266,6 +266,10 @@ Optional parameters to customize the camera settings.
 | saveToPhotoAlbum | <code>Boolean</code> |  | Save the image to the photo album on the device after capture. |
 | popoverOptions | <code>[CameraPopoverOptions](#module_CameraPopoverOptions)</code> |  | iOS-only options that specify popover location in iPad. |
 | cameraDirection | <code>[Direction](#module_Camera.Direction)</code> | <code>BACK</code> | Choose the camera to use (front- or back-facing). |
+| customCameraContainer | <code>string</code> |  | Browser-only option, specify the id of custom camera's container element.  |
```

Comment:
[1] IMO, I think it would be better if these new options accept either a `query` string or `DOMElement`.

* `Query` string would give more flexibility in how the element is discovered vs an ID only. In this case, we would need to use the `document.querySelector` method instead of the `getElementById`.

* `DOMElement` could also be a viable option.

Reply:
`customCameraContainer` sounds like is something you need a specific `DOMElement`, so allowing flexibility of `querySelector` I think promotes bad coding practice.

If you need a specific element, the element should be tagged by an ID as to guarantee that you will receive the expected `DOMElement`. Using `querySelector` makes code brittle by picking the first occurrence of the selected term, which may not be the node you want.

The YouTube JS API[1] for example does something similar where it needs to insert HTML in a specific node of your choice. Their API signature accepts either an id, or the `HTMLElement` itself.

So I would prefer to keep `customCameraContainer` accepting an `id` rather than a query string. For flexibility, it could also optionally accept a DOM node and it would be up to the developer to find that reference as they see fit (which they could use `querySelector` if they wanted to, or it could be a fresh node that isn't even in the DOM tree yet (via `document.createElement`), etc.

[1] https://developers.google.com/youtube/iframe_api_reference#Loading_a_Video_Player

Reply:
I believe this PR is actually no longer valid.

There were some discussion after the PR creation and a decision that users should maybe use WebRTC camera functionality over this plugin. After that, this PR development was stopped. Maybe this PR can be closed.

If memory serves, iOS has some limitation/problems with WebRTC camera functionality so the plugin will still be needed. It can be conditionally wrapped to decided if it use WebRTC over Plugin.

Reply:
Must be this limitation:

> Does not work in standalone running ("installed") PWAs, getUserMedia returns no video input devices in UIWebView or WKWebView, but only directly in Safari.

https://caniuse.com/#search=getUserMedia